workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -474,6 +475,10 @@ private JSONObject _transformIssueFields(JSONObject issueFieldsJSONObject) {
 			"affectedVersionsDetails",
 			issueFieldsJSONObject.optString(
 				_jiraSecurityVulnerabilityFieldAffectedVersionsDetails)
+		).put(
+			"affects",
```

Comment:
i think this should be sorted to be after `affectedVersions`?

Reply:
Done!